### PR TITLE
Better tailwind config

### PR DESCRIPTION
**Pull request by f-elix**
The negative spacing values in the current version of the config are redundant, since Tailwind already generates a negative value class for each value. I also added most percentages. I added all of these to maxWidth, minWidth, maxHeight, minHeight and inset. I also removed the old inset object in the "extend" property, since it is not needed anymore with this update.

Any objections?

### CODE REVIEW

**Review by fhamon — APPROVED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -46,28 +46,27 @@ module.exports = {
 				50: 'rgba(255,255,255,0.5)',
 				20: 'rgba(255,255,255,0.2)',
 				0: 'rgba(255,255,255,0)'
-			},
-			
+			}
```
Comment (dominic-blain) [inline tailwind.config.js:49]: why? :(

File: tailwind.config.js [hunk 2]
```diff
@@ -76,81 +75,94 @@ module.exports = {
 			0: '0',
 			px: '1px',
 			pico: '1rem',
-			nano: '3rem',
+			nano: '4rem',
 			micro: '8rem',
 			thinnest: '12rem',
 			thinner: '16rem',
 			thin: '20rem',
+			'thin+': '24rem',
```
Comment (dominic-blain) [inline tailwind.config.js:83]: "+" items in the scale should not exist. They are a last resort if more values are needed but should be added per project.

File: tailwind.config.js [hunk 3]
```diff
@@ -76,81 +75,94 @@ module.exports = {
...
+			'1/20': '5%',
+			'3/20': '15%',
+			'7/20': '35%',
+			'9/20': '45%',
+			'11/20': '55%',
+			'13/20': '65%',
+			'17/20': '85%',
+			'19/20': '95%',
+			full: '100%',
+			'screen-1/4': '25vh',
```
Comment (dominic-blain) [inline tailwind.config.js:128]: The choice of vh for "screen" here is arbitrary. We could need vw or vmin. Should probably be `'1/4vh' : '25vh'` and then could also have `'1/4vw' : '25vw'`

File: tailwind.config.js [hunk 4]
```diff
@@ -76,81 +75,94 @@ module.exports = {
...
+			'screen-3/4': '75vh'
 		},
 		maxWidth: (theme) => ({
 			none: 'none',
-			full: '100%',
-			'9/10': '90%',
-			'3/4': '75%',
-			'1/2': '50%',
 			...theme('spacing'),
+			...theme('screens'),
```
Comment (dominic-blain) [inline tailwind.config.js:135]: Screens is already there (line below)

File: tailwind.config.js [hunk 5]
```diff
@@ -76,81 +75,94 @@ module.exports = {
...
 			...except('touch', theme('screens'))
 		}),
 		minWidth: (theme) => ({
 			none: 'none',
-			full: '100%',
 			...theme('spacing'),
+			...theme('screens'),
```
Comment (dominic-blain) [inline tailwind.config.js:141]: Screens is already there (line below)

File: tailwind.config.js [hunk 6]
```diff
@@ -76,81 +75,94 @@ module.exports = {
...
+			...except('touch', theme('screens'))
+		}),
+		maxHeight: (theme) => ({
+			none: 'none',
+			...theme('spacing'),
+			...theme('screens'),
```
Comment (dominic-blain) [inline tailwind.config.js:147]: Screens is already there (line below)

File: tailwind.config.js [hunk 7]
```diff
@@ -76,81 +75,94 @@ module.exports = {
...
 			...except('touch', theme('screens'))
 		}),
+		minHeight: (theme) => ({
+			none: 'none',
+			...theme('spacing'),
+			...theme('screens'),
```
Comment (dominic-blain) [inline tailwind.config.js:153]: Screens is already there (line below)

File: tailwind.config.js [hunk 8]
```diff
@@ -76,81 +75,94 @@ module.exports = {
...
+			...except('touch', theme('screens'))
+		}),
+		inset: (theme) => ({
+			none: 'none',
+			...theme('spacing'),
+			...theme('screens'),
```
Comment (dominic-blain) [inline tailwind.config.js:159]: Screens is already there (line below)

File: tailwind.config.js [hunk 9]
```diff
@@ -76,81 +75,94 @@ module.exports = {
...
 			...except('touch', theme('screens'))
 		}),
+		minHeight: (theme) => ({
+			none: 'none',
+			...theme('spacing'),
+			...theme('screens'),
+			...except('touch', theme('screens'))
+		}),
+		inset: (theme) => ({
+			none: 'none',
```
Comment (dominic-blain) [inline tailwind.config.js:157]: Let's add auto too

File: tailwind.config.js [hunk 10]
```diff
@@ -76,81 +75,94 @@ module.exports = {
...
+			'13/20': '65%',
+			'17/20': '85%',
+			'19/20': '95%',
+			full: '100%',
+			'screen-1/4': '25vh',
+			'screen-1/2': '50vh',
+			'screen-3/4': '75vh'
 		},
 		maxWidth: (theme) => ({
 			none: 'none',
```
Comment (dominic-blain) [inline tailwind.config.js:133]: Let's add auto too

File: tailwind.config.js [hunk 11]
```diff
@@ -76,81 +75,94 @@ module.exports = {
...
 		}),
 		minWidth: (theme) => ({
 			none: 'none',
-			full: '100%',
 			...theme('spacing'),
+			...theme('screens'),
+			...except('touch', theme('screens'))
+		}),
+		maxHeight: (theme) => ({
+			none: 'none',
```
Comment (dominic-blain) [inline tailwind.config.js:145]: Let's add auto too

File: tailwind.config.js [hunk 12]
```diff
@@ -76,81 +75,94 @@ module.exports = {
...
+			...theme('spacing'),
+			...theme('screens'),
 			...except('touch', theme('screens'))
 		}),
+		minHeight: (theme) => ({
+			none: 'none',
```
Comment (dominic-blain) [inline tailwind.config.js:151]: Let's add auto too

**Review by dominic-blain — CHANGES_REQUESTED** [review]
Nice cleanup! Some fixes required and we're good to go

File: tailwind.config.js (hunk 1 above)
Comment (f-elix) [reply tailwind.config.js:49]: Oops, will remove :)

File: tailwind.config.js (hunk 2 above)
Comment (f-elix) [reply tailwind.config.js:83]: That's right, my copy/paste mistake! Will remove.

File: tailwind.config.js (hunk 3 above)
Comment (f-elix) [reply tailwind.config.js:128]: Yes, already removed it in the last commit.

**Review by dominic-blain — CHANGES_REQUESTED** [review]

**Review by dominic-blain — APPROVED** [review]

**Pull request merged**